docs/setup.rst

```diff
@@ -94,23 +94,107 @@ The following sub-sections provide guidance on how to install the recommended to
 - **Linux** TODO
 - **macOS** TODO
 
+.. _setup_zlib:
+
+zlib
+----
+
+:zlib:`zlib<>` is a common compression library.
+It's important to use a 64-bit version of the library as the 32-bit is not compatible with libCellML.
+
+- **Windows** On Windows, it is easiest to install using the packaged version available from the :opencmiss_zlib_repo:`OpenCMISS zlib releases page<>` on GitHub.
+  You are welcome to build your own version if you would rather, but please make sure it is a 64-bit implementation.
+  To build :zlib:`zlib<>` from source refer to the next sub-section.
+- **Linux** For Debian based systems, zlib can be installed using :code:`sudo apt install zlib1g-dev`.
+- **macOS** zlib is already installed on macOS, so no further action is required on that platform.
+
+Building from source
+++++++++++++++++++++
+
+To create a zlib library suitable for libCellML, we need to download, configure, build, and install it.
+
+To download zlib clone the repository https://github.com/OpenCMISS-Dependencies/zlib::
+
+  git clone https://github.com/OpenCMISS-Dependencies/zlib -b v1.2.3
+
+Create a build directory and change into it::
+
+  mkdir build-zlib
+  cd build-zlib
+
+Create the zlib library build instructions using CMake::
+
+  cmake -DCMAKE_INSTALL_PREFIX=<ZLIB_INSTALL_DIR> ..\zlib
+
+Where :code:`<ZLIB_INSTALL_DIR>` is to be replaced by a path on your computer.
+Something like::
+
+  C:\Users\Andre\libcellml\libraries
+
+if your username was :code:`Andre`.
+
+We can now build the library with the command::
+
+  cmake --build . --config Release
+
+The last thing we do is to install the library with the command::
+
+  cmake --build . --target install --config Release
+
 .. _setup_libxml2:
 
-LibXml2
+libxml2
 -------
-:libxml2:`LibXml2<>` is a parser and toolkit for manipulating XML files and text.  
-It's important to use a 64-bit version of the library as the 32-bit is not compatible with libCellML.  
 
-- **Windows** On Windows, it's easiest to install using the packaged version available from :opencmiss_repo:`the OpenCMISS site<>` .
-  You're welcome to build your own version if you'd rather, but please make sure it's a 64-bit implementation.  
-  Source code for building it yourself can be downloaded from the :libxml2:`LibXml2<>` site.
-- **Linux** LibXml2 can be installed using :code:`sudo apt install libxml2-dev`.
-- **macOS** LibXML2 is already installed on macOS, so no further action is required on that platform.  
+:libxml2:`libxml2<>` is a parser and toolkit for manipulating XML files and text.
+It's important to use a 64-bit version of the library as the 32-bit is not compatible with libCellML.
```

Comment:
`It's` should still be replaced with `It is`.

Reply:
8d210dc